auth-web/tests/unit/services/task.service.spec.ts

```diff
@@ -21,6 +20,25 @@ const task: Task[] = [{
   'type': 'Wills Registry'
 }]
 
+jest.mock('axios', () => {
```

Comment:
The old mock would fail, if ran concurrently.. would actually try to login to auth-api-dev.